spidermon/contrib/validation/cerberus/validator.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+from cerberus.validator import (
+    Validator as cerberus_validator,
+    DocumentError,
+    SchemaError
+)
+from spidermon.contrib.validation.validator import Validator
+from .translator import CerberusMessageTranslator
+
+
+class CerberusValidator(Validator):
+    default_translator = CerberusMessageTranslator()
+    name = "Cerberus"
+
+    def __init__(self, schema, translator=None, use_default_translator=True):
+        super(CerberusValidator, self).__init__(
+            translator=translator,
+            use_default_translator=use_default_translator
+        )
+        self._schema = schema
+
+
+    def _validate(self, data, strict=False):
+        try:
+            validator = cerberus_validator(self._schema)
+            validator.validate(data)
+            errors = validator.errors
```

Comment:
No need for this variable.
Just `for field_name, error in validator.errors.items():`

Reply:
Good addition, coming right up.